### DISCUSSION TITLE
[Config Support]: Possible to target multiple MQTT brokers?

### DISCUSSION THREAD

**Discussion by kavejo** [blakeblackshear/frigate#15691] [Config Support]
### Describe the problem you are having

I’m in the process of rebuilding my home server and during the transition phase I will have 2 HomeAssistant instances and 2 Frigate instances, until the full configuration is completed.

Is it possible for any given Frigate instance to target multiple MQTT targets. So that both HA can read from the queue events from any of the Frigate instances?

When looking at the configuration I saw a single “mqtt:” section.

### Version

stable

### Frigate config file

```yaml
N/A
```

### Relevant Frigate log output

```shell
N/A
```

### Relevant go2rtc log output

```shell
N/A
```

### Frigate stats

```json
N/A
```

### Operating system

HassOS

### Install method

HassOS Addon

### docker-compose file or Docker CLI command

```yaml
N/A
```

### Object Detector

Coral

### Screenshots of the Frigate UI's System metrics pages

N/A

**Comment by NickM-27** [comment]
I don't see a need for you to have 2 mqtt instances. One should work for both as long as the other Frigate instance has a different topic and client id

**Reply by kavejo** [reply]
Makes sense, that’s a possible set-up.

I was aiming to host one Mosquitto instance on one home server running HassOS and another one on the new home server that will be running a Home Assistant Container. And each will have its own Feigate container associated. 

My goal is to have both Frigate updating both HA so that I can avoid missing notifications until I move the client devices from one instance to the other, and I can compare the different detection (the “old” instance uses Coral and h.264 streams, the new one will be using Tensor and h.265 streams).

I appreciate I could just achieve this by using a single MQTT broker and having all 4 entities reading/writing from/to the same, using different topics, but I’d prefer - if possible - to run separate instances during the transition phase.
